### ISSUE TITLE
New links get the same address as peeringdb

### ISSUE THREAD

**Issue by pcfriek1987** [thatmattlove/hyperglass#145]
I added the peeringdb link on the left which works fine, but also tried adding a second link on the right, which still has the peeringdb link even though it has a different value.

  links:
    - order: 0
      show_icon: true
      side: left
      title: PeeringDB
      url: https://www.peeringdb.com/asn/{primary_asn}
    - order: 5
      side: right
      title: Website
      url: https://google.com

I replaced the website with a generic one since this is my private github account :)

**Comment by checktheroads** [comment]
Hmm, I'm not able to reproduce this. My config:

```yaml
web:
    links:
        - title: PeeringDB
          order: 2
          url: https://peeringdb.com
        - title: Another Link
          order: 1
          url: https://www.google.com
          side: right
        - title: Yet Another Link
          order: 3
          url: https://www.cloudflare.com
          show_icon: false
```

When I have that in my config and start hyperglass (with `--build-ui` or `-b`), I see the links with the correct URLs. Can you try running `hyperglass build-ui` manually and see if that makes a difference?

**Comment by pcfriek1987** [comment]
I can see it correctly in the build ui but even after a manual build it still goes to the wrong link.

 [{'order': 0, 'show_icon': True, 'side': 'left', 'title': 'PeeringDB', 'url': 'https://www.peeringdb.com/asn/{primary_asn}'}, {'title': 'NFOrce Website', 'order': 5, 'side': 'right', 'url': 'https://www.nforce.com/'}]

Nvm seeing the website here, guess it's easier like this.

**Comment by thatmattlove** [comment]
Is this still happening for you? Even when I go to your LG, I see the correct links:

<img width="1195" alt="Screen Shot 2021-07-03 at 10 15 10 PM" src="https://user-images.githubusercontent.com/42724207/124374014-73998c00-dc4c-11eb-911c-653620a93640.png">

<img width="335" alt="Screen Shot 2021-07-03 at 10 15 18 PM" src="https://user-images.githubusercontent.com/42724207/124374016-76947c80-dc4c-11eb-98b6-d5a07f0e1557.png">

**Comment by pcfriek1987** [comment]
I actually still do indeed, but it seems to be a firefox issue now that I can see as in Chrome it displays them correctly.

**Comment by thatmattlove** [comment]
I can confirm that this issue exists:

1. In Firefox only
2. On hyperglass 1.0.4
3. Only if you use external links on both the left and right side

That being said, the issue appears to be fixed on the v2.0.0 branch, for reasons unknown. More than likely, a dependency update fixed the underlying issue accidentally. 🤷‍♂️

**Comment by pmawsonau** [comment]
Hi
Notice this issue on 1.0.4 on Firefox when all links are left. (https://lg.vocus.network)
When is v2.0.0 expected to be stable/main release?
Cheers,
Phil
